www/css/app.css

```diff
@@ -64,6 +64,10 @@
     padding-left: 1px !important;
 }
 
+.dragging-over * {
+    pointer-events: none !important;
```

Comment:
Is `!important` necessary? Not a problem if it's needed, but did you test without?

Reply:
I think I did test, but the issue at the time was something else (not 100% sure on that) and I never changed it back. It may well be unnecessary

Reply:
OK, if it turns out not to be necessary, best to remove it.